server/webapp/WEB-INF/rails.new/app/assets/new_stylesheets/shared/_go-variables.scss

```diff
@@ -14,14 +14,115 @@
  * limitations under the License.
  */
 
-$go-primary: #94399e;
-$go-secondary: #40d9da;
-$go-tertiary: #f5a623;
+$go-primary: #b66fc2;
+$go-secondary: #00bfa5;
+$go-tertiary: #ffd54f;
 $go-white: #fff;
 $light-grey: #e2e2e2;
 $grey: #535353;
 $dark-grey: #1e1e1e;
-
+$txt-color: #333;
 $border-color: #ccc;
+$go-links: #0277bd;
 
 $global-border-radius: 3px;
+
+//table
+
+$table-header-bg: #b9f6ca;
+$body-background: #efefef;
```

Comment:
icon_breadcrumb_arrow.svg was created as a part of the dark theme earlier and not deleted. Removed it